shared-bindings/socketpool/__init__.c

```diff
@@ -35,6 +35,9 @@
 //| """
 //| The `socketpool` module provides sockets through a pool. The pools themselves
 //| act like CPython's `socket` module.
+//|
+//| For more infomration about the `socket` module, see the CPython documentation:
+//| https://docs.python.org/3/library/socket.html
```

Comment:
```suggestion
//| |see_cpython_module| :mod:`cpython:random`
```

Reply:
@tekktrik Will this work? (I noticed `information` had a typo anyway.)

Reply:
Yup!  Tested locally, it's the same trick used in the `*.rst` files!  I'll fix the typo.

Reply:
Yup!  It just adds the hyperlink.  Seemed better than using the note, because the two modules have different names.  I can use the shortcut note if you think that's better though.

Reply:
Dan's suggestion of changing it from "socket" to "random" seems dubious :stuck_out_tongue_winking_eye:

Reply:
Dubiousness is my specialty.